src/pds/registrysweepers/legacy_registry_sync/solr_doc_export_to_opensearch.py

```diff
@@ -0,0 +1,97 @@
+import os
+import logging
+from datetime import datetime
+
+log = logging.getLogger(__name__)
+
+NODE_FOLDERS = {
+    "atmos": "PDS_ATM",
+    "en": "PDS_ENG",
+    "geo": "PDS_GEO",
+    "img": "PDS_IMG",
+    "naif": "PDS_NAIF",
+    "ppi": "PDS_PPI",
+    "rings": "PDS_RMS",
+    "rs": "PDS_RS",
+    "sbn": "PDS_SBN",
+}
+
+
+class MissingIdentifierError(Exception):
+    pass
+
+
+def pds4_id_field_fun(doc):
+    if 'version_id' in doc:
+        return doc['identifier'] + '::' + doc['version_id'][-1]
+    elif 'identifier' in doc:
+        return doc['identifier']
+    else:
+        raise MissingIdentifierError()
+
+
+def get_node_from_file_ref(file_ref: str):
+    file_path = os.fspath(file_ref)
+    path = os.path.normpath(file_path)
+    dirs = path.split(os.sep)
+    return NODE_FOLDERS.get(dirs[4], "PDS_EN")
+
+
+class SolrOsWrapperIter:
+    def __init__(self, solr_itr, es_index, found_ids=None):
+        self.index = es_index
+        self.type = "_doc"
+        self.id_field_fun = pds4_id_field_fun
+        self.found_ids = found_ids
+        self.solr_itr = iter(solr_itr)
+
+    def __iter__(self):
+        return self
+
+    def solrDoc_to_osDoc(self, doc):
```

Comment:
It looks like the pre-commit hooks aren't working in this module? There are numerous missing docstrings that should've been flagged 🤷

Reply:
I might have missed to install pre-commit for this repository. I will do that now.